e2e/config/azure.go

```diff
@@ -345,3 +475,19 @@ func DefaultRetryOpts() policy.RetryOptions {
 		},
 	}
 }
+
+func replicatedToCurrentRegion(version *armcompute.GalleryImageVersion) bool {
```

Comment:
since this member function isn't part of a GalleryImageVersion class, I would renamed it to be more explicit
nit: `isImageVersionReplicatedToCurrentRegion`

Reply:
I don't have a strong opinion on whether we should prefix all `bool` function with `Is`.

But I don't think we're doing it currently.